src/apps/checkout/index.md

```diff
@@ -1,19 +1,70 @@
-# Checkout
+<script setup>
```

Comment:
I'll move those components to the `developer-documentation-vitepress` and make them globally available so we don't need to manually re-import components every time we need them. Let's do that tomorrow.